extensions/htlc/besu-test-eth/src/test/solidity/hash-time-lock-contract/DemoHelpers.json

```diff
@@ -0,0 +1,4229 @@
+{
```

Comment:
@AzaharaC Question regarding the entire packages/cactus-test-plugin-htlc-eth-besu/src/test/solidity/hash-time-lock-contract/ directory. Is this any different from the contract code that's in the HTLC plugin? If not, then the tests could just import the JSON artifacts directly from there and this folder here can be deleted to avoid duplicating too much.